core/tests/webdriverio_desktop/voiceoverUploadFeatures.js

```diff
@@ -135,6 +137,7 @@ describe('Voiceover upload features', function() {
         'test_audio_1_en.mp3');
       await explorationEditorTranslationTab.saveAudioRecord();
       await explorationEditorTranslationTab.playAudioRecord();
+      await action.waitForAutosave();
```

Comment:
I'm wondering why we need `waitForAutosave` after play and not after `save`? (Can we move waitForAutosave inside play/saveAudioRecord function? In general, actions are performed inside utils file.)

Reply:
Ah, is it because "we are doing refresh after it and want to avoid incomplete save call"? If yes then it's better to wait after `saveAudioRecord` or `confirmAudioRecord`.

Reply:
Done

Reply:
Sorry for the confusion, can we move `await action.waitForAutosave();`inside `saveAudioRecord` & `confirmAudioRecord`? This way `saveAudioRecord` will be a complete action and the developer doesn't have to worry about writing `waitForAutosave` after every  `saveAudioRecord`.

Reply:
Done